resources/java_memory_assistant/bin/clean-up.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
```

Comment:
The [AWS SDK](https://github.com/aws/aws-sdk-java/blob/master/LICENSE.txt) is Apache licensed (I'm going to guess that all of them are since the Java one is), which is suitable for inclusion.